teacher.rb

```diff
@@ -0,0 +1,15 @@
+require './person'
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(specialization, name = 'Unknown')
+    super(name)
+    self.name = name
+    @specialization = specialization
+  end
```

Comment:
- [ ] For class Teacher, while making a call to the `parent class`, kindly ensure that all the necessary `parameters `are initialized alongside with **`name`**.

## Project Requirement
<img width="494" alt="Untitled" src="https://github.com/Ochiengsteven/school-library-oop/assets/103386183/ebd7deaf-bcda-4c83-baf3-8cb3911dc86d">

```suggestion
  def initialize(specialization, name = 'Unknown')
    super(age, name, parent_permission)
    self.name = name
    @specialization = specialization
  end
```